forte/data/ontology/top.py

```diff
@@ -268,17 +228,26 @@ class Link(BaseLink):
         child: the child entry of the link.
     """
     # this type Any is needed since subclasses of this class will have new types
-    ParentType: Any = Entry
-    ChildType: Any = Entry
+
+    parent_type: str
+    child_type: str
+    parent: Optional[Entry]
+    child: Optional[Entry]
```

Comment:
This is changing the interface. Previously `Link.parent` and `Link.child` are `tid` instead of entry object.